.ci/validate_notebooks.py

```diff
@@ -59,13 +59,25 @@ def prepare_test_plan(test_list, ignore_list):
     return statuses
 
 
+def clean_test_artefacts(before_test_files, after_test_files):
+    for file_path in after_test_files:
+        if file_path in before_test_files or not file_path.exists():
+            continue
+        if not file_path.is_file():
+            file_path.unlink()
+        else:
+            file_path.rmdir()
```

Comment:
Will it work for directories with files inside?

Reply:
replaced with another command